doc/installation.rst

```diff
@@ -427,11 +427,14 @@ that SmartSim needs already built into it. Users can follow these instructions
 to get a working SmartSim build with PyTorch for GPU on Summit.
 
 .. code-block:: bash
-
   # setup Python and build environment
   module load open-ce
-  conda create -p /ccs/home/<USERNAME>/.conda/envs/smartsim --clone open-ce-1.2.0-py38-0
+  conda create -p /ccs/home/$USER/.conda/envs/smartsim --clone open-ce-1.2.0-py38-0
   conda activate smartsim
+  # fix broken cmake module
+  pip uninstall cmake
+  pip install cmake
+  conda install git-lfs make -y
```

Comment:
I think we should add ``git-lfs install`` here.

Reply:
I've added it to the instructions.